db/Cargo.toml

```diff
@@ -4,13 +4,15 @@ version = "0.0.1"
 
 [dependencies]
 error-chain = "0.8.0"
+itertools = "0.5.9"
 lazy_static = "0.2.2"
 # TODO: don't depend on num and ordered-float; expose helpers in edn abstracting necessary constructors.
 num = "0.1.35"
 ordered-float = "0.3.0"
+time = "0.1.35"
```

Comment:
Is there anyway to keep these deps (and other modules) in sync with the top-level dependencies?